pydsfapi/tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py{35,36,37,38,39}
+
+[testenv]
+deps =
+    check-manifest
+    black
+    mypy
+    flake8
+    pytest
+commands =
+    check-manifest --ignore 'tox.ini,docs/**,examples/**,tests/**'
+    python setup.py check -m -s
+    black --check --target-version py36 .
```

Comment:
Again, if DuetPi is still running 3.5 (I hope it's not) this needs to check for compatibility with 3.5.

Reply:
see above discussion - I think we are fine here.
Raspbian Buster comes with python 3.7, and the latest https://pkg.duet3d.com/DuetPi-lite.zip is Buster.